.travis.yml

```diff
@@ -11,6 +11,11 @@ php:
 os:
   - linux
 
+env:
+  matrix:
+    - PREFER_LOWEST="--prefer-lowest"
```

Comment:
Please also use `--prefer-lowest --prefer-stable` here to avoid alpha/beta/RC usage etc.